<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>708</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>708</width>
    <height>433</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Iceshell</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DdLabel" name="label_file_input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/ui_bg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QMenu" name="menuExtension">
     <property name="title">
      <string>Extension</string>
     </property>
     <addaction name="actionSame"/>
     <addaction name="actionjpg_2"/>
     <addaction name="actionpng_2"/>
     <addaction name="actionwebp_2"/>
    </widget>
    <widget class="QMenu" name="menuFolder">
     <property name="title">
      <string>Folder</string>
     </property>
     <addaction name="actionSame_as_input_2"/>
     <addaction name="actionSelect_2"/>
    </widget>
    <addaction name="menuExtension"/>
    <addaction name="menuFolder"/>
   </widget>
   <widget class="QMenu" name="menuUpscale">
    <property name="title">
     <string>Upscale</string>
    </property>
    <addaction name="action1_3"/>
    <addaction name="action2_3"/>
    <addaction name="action3_3"/>
    <addaction name="action4_2"/>
   </widget>
   <widget class="QMenu" name="menuDenoise">
    <property name="title">
     <string>Denoise</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action0_2"/>
    <addaction name="action1_4"/>
    <addaction name="action2_4"/>
    <addaction name="action3_4"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_2"/>
   </widget>
   <widget class="QMenu" name="menuUpscaler">
    <property name="title">
     <string>Upscaler</string>
    </property>
    <addaction name="actionReal_CUGAN"/>
    <addaction name="actionReal_ESRGAN"/>
    <addaction name="actionWaifu2X"/>
   </widget>
   <addaction name="menuUpscale"/>
   <addaction name="menuDenoise"/>
   <addaction name="menuOutput"/>
   <addaction name="menuUpscaler"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action1x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1x</string>
   </property>
  </action>
  <action name="action2x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2x</string>
   </property>
  </action>
  <action name="action3x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3x</string>
   </property>
  </action>
  <action name="action4x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4x</string>
   </property>
  </action>
  <action name="action1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </action>
  <action name="action0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionsame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Same as input</string>
   </property>
  </action>
  <action name="actionjpg">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>jpg</string>
   </property>
  </action>
  <action name="actionpng">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>png</string>
   </property>
  </action>
  <action name="actionwebp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>webp</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSame_as_input">
   <property name="text">
    <string>Same as input</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select...</string>
   </property>
  </action>
  <action name="actionSame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Same as Input</string>
   </property>
  </action>
  <action name="actionjpg_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>.jpg</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </action>
  <action name="action0_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionSame_as_input_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Same as Input</string>
   </property>
  </action>
  <action name="actionpng_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>.png</string>
   </property>
  </action>
  <action name="actionwebp_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>.webp</string>
   </property>
  </action>
  <action name="actionSelect_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select...</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReal_CUGAN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Real-CUGAN</string>
   </property>
  </action>
  <action name="actionReal_ESRGAN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Real-ESRGAN</string>
   </property>
  </action>
  <action name="actionWaifu2X">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Waifu2x</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DdLabel</class>
   <extends>QLabel</extends>
   <header>ddlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bg.qrc"/>
 </resources>
 <connections/>
</ui>
